src/components/AppSidebar/AppSidebar.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <v-navigation-drawer
     class="app-sidebar"
-    floating
     absolute
-    width="440"
-    :value="appNavigationOpen"
     disable-route-watcher
     disable-resize-watcher
+    floating
+    stateless
```

Comment:
I think when using `stateless` you can get rid of the other 'disable watcher' props

Reply:
Removed.